datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -539,7 +539,7 @@ private void validateMediumDutySdxEnablement(SdxClusterRequest sdxClusterRequest
         ValidationResultBuilder validationBuilder = new ValidationResultBuilder();
         if (SdxClusterShape.MEDIUM_DUTY_HA.equals(sdxClusterRequest.getClusterShape())) {
             boolean mediumDutySdxEntitlementEnabled = entitlementService.mediumDutySdxEnabled(Crn.safeFromString(environment.getCreator()).getAccountId());
-            if (!mediumDutySdxEntitlementEnabled) {
+            if (!(AWS.equalsIgnoreCase(environment.getCloudPlatform()) || mediumDutySdxEntitlementEnabled)) {
                 validationBuilder.error("Provisioning a medium duty data lake cluster is not enabled for this account. " +
```

Comment:
Update this to state that "medium duty data lake cluster on **Azure** is not enabled"

Reply:
Wouldn't we want to make this general for any of the cloud platforms? Would it be better to say "...medium duty data lake cluster is not enabled for this cloud platform"?

Reply:
Why don't you spit the difference, and do something like:
```
validationBuilder.error(String.format("Provisioning a medium duty data lake cluster is not enabled for %s. " +
        "Contact Cloudera support to enable CDP_MEDIUM_DUTY_SDX entitlement for the account.", environment.getCloudPlatform()));
```

Reply:
Sounds good I'll do that.